mcs/class/Mono.Options/Test/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think for mono code we always use nunitlite, not sure we need to be different here